com.unity.visualeffectgraph/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Resize custom operator (multiply, add...) to the minimum size when changing input types
 - Show opened VFX asset in inspector when clicking in a void area and unselect node when VFX graph loose focus
 - Disabled text inputs were unreadable [Case 1387237](https://issuetracker.unity3d.com/product/unity/issues/guid/1387237/)
+- In `Output Particle` blocks, shader graph exposed properties order is consistent with shader graph blackboard
```

Comment:
Not sure why this case is public but you can link it here: https://issuetracker.unity3d.com/product/unity/issues/guid/1398256/

Reply:
The link seems broken ...

Reply:
Strange, it's marked as public here https://fogbugz.unity3d.com/f/cases/1398256/